### PR TITLE
🔧 libs/ui build error

### LINKED ISSUE #8251: translates break historie stories
### Is there an existing issue?

- [X] I have searched the existing issues

### What happened?

Screenshot 2023-11-25 at 21 55 43

### Please reproduce in steps

1. Run stories
```sh
cd libs/ui
pnpm run story:dev
```
2. Add `is-lewd` props to MediaItem.story
```vue
<Variant title="image">
  <MediaItem
    src="https://image-beta.w.kodadot.xyz/ipfs/bafybeig6clbsbabxkku3m4furbj4klayyx7ovwtcocmgwqmv4nxllnbxxy"
    :is-lewd="true" />
</Variant>
```
3. Open http://localhost:6006/story/src-components-mediaitem-mediaitem-story-vue?variantId=src-components-mediaitem-mediaitem-story-vue-0

### Expected Behavior

No errors

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

None

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

--

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/ksm/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by roiLeo**
-related with #8251 

Who made `TheImage` component? name is misleading & component import is bad.

Capture d’écran 2023-11-27 à 10 55 23 AM

### CODE REVIEW

**Review by daiagi — APPROVED** [review]

**Comment by vikiival** [comment]
Maybe rollback CODEOWNERS?

**Comment by roiLeo** [comment]
> Maybe rollback CODEOWNERS?

why?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,2 @@
-*       @kodadot/code-review-guild
+*			@kodadot/code-review-guild
+/libs/ui	@roiLeo
```
Comment (vikiival) [inline .github/CODEOWNERS:2]:
```suggestion
/libs/ui	@kodadot/internal-dev 
```

**Review by vikiival — APPROVED** [review]

**Comment by preschian** [comment]
anyway, can we capture the error on ci with some commands?
maybe put `pnpm -F brick story:build` on ci

File: .github/CODEOWNERS (hunk 1 above)
Comment (roiLeo) [reply .github/CODEOWNERS]: then this line is unnecessary, I wanted to be added as reviewer each code change in libs/ui

**Comment by roiLeo** [comment]
> anyway, can we capture the error on ci with some commands? maybe put `pnpm -F brick story:build` on ci

https://github.com/kodadot/nft-gallery/blob/main/.github/workflows/build.libs-ui.yml
or #4464

**Comment by roiLeo** [comment]
> is there any way to customize the linter to avoid such problems in the future? 

move project outside repo

> We don't need to do explicit imports in the main project, but in ui kit we do, because of this problem.

can be fixed if we migrate to Vue histoire to Nuxt histoire

> I would like to solve such a problem in a centralized way

Let us know how you would proceed before starting anything

**Comment by roiLeo** [comment]
> $t('lewd.explicit')

same as https://github.com/kodadot/nft-gallery/pull/8124#discussion_r1404636293 we don't need i18n in UI package

**Comment by roiLeo** [comment]
> About the naming of this component, this is a common Image component, I named it along the lines of `TheButton`. What are some other options for naming the common component of the image?

I don't think it's a good idea to prefix with `The*`, if it's common then it should be `Common*` like `CommonPrimaryButton`, `CommonInvertedButton` etc.. we can use Nuxt auto import feature for better naming (`/components/common/*`)

**Pull request merged**